src/qt/editor/inviwomainwindow.cpp

```diff
@@ -957,6 +1014,38 @@ void InviwoMainWindow::saveWorkspaceAsCopy() {
     saveWindowState();
 }
 
+void InviwoMainWindow::showWelcomeScreen() {
+    if (!welcomeWidget_) {
+        welcomeWidget_ = new WelcomeWidget(this, centralWidget_);
+
+        centralWidget_->setUpdatesEnabled(false);
+        centralWidget_->setCurrentIndex(0);
+        centralWidget_->insertTab(0, welcomeWidget_, "Get Started");
+        centralWidget_->setUpdatesEnabled(true);
+
+        QToolButton* closeBtn = new QToolButton();
+        closeBtn->setIcon(QIcon(":/stylesheets/images/close.png"));
+        closeBtn->setObjectName("dockBtn");
+        QObject::connect(closeBtn, &QToolButton::clicked, this, [&]() { hideWelcomeScreen(); });
+
+        centralWidget_->tabBar()->setTabButton(0, QTabBar::RightSide, closeBtn);
+    }
+    centralWidget_->setCurrentIndex(centralWidget_->indexOf(welcomeWidget_));
```

Comment:
Use raise?

Reply:
will that work within a QTabWidget?

Reply:
Think I use that in the in the docked ones at least. for the propertylistwidget...